package.json

```diff
@@ -167,7 +167,7 @@
     "ember-simple-auth": "^1.6.0",
     "ember-sinon": "^5.0.0",
     "ember-sinon-qunit": "^5.0.0",
-    "ember-sortable": "^1.11.2",
+    "ember-sortable": "2.2",
```

Comment:
Check collections to see if it's breaking the already broken contributors widget there.